src/Core/Services/Implementations/OrganizationService.cs

```diff
@@ -1068,21 +1068,54 @@ private async Task SendInviteAsync(OrganizationUser orgUser, Organization organi
             IUserService userService)
         {
             var orgUser = await _organizationUserRepository.GetByIdAsync(organizationUserId);
+            
             if (orgUser == null)
             {
                 throw new BadRequestException("User invalid.");
             }
-
-            if (orgUser.Status != OrganizationUserStatusType.Invited)
+            
+            if (!CoreHelpers.UserInviteTokenIsValid(_dataProtector, token, user.Email, orgUser.Id, _globalSettings))
             {
-                throw new BadRequestException("Already accepted.");
+                throw new BadRequestException("Invalid token.");
             }
-
+            
```

Comment:
Remove extra white-space

Reply:
Same here, it's just the blank spaces on the empty line that need to be removed, not the line itself.